lib/logstash/outputs/dynatrace.rb

```diff
@@ -108,7 +112,6 @@ def multi_receive(events)
           end
         rescue StandardError => e
           @logger.error("Unknown error raised", :error => e.inspect)
-          raise e
```

Comment:
Shouldn't the extra clause for well-known errors above be sufficient while unknown errors could still be rethrown?

Reply:
If we throw it crashes the whole process which is basically never wanted unless there is an actual code error being run into.

Reply:
Actually I think you're right we do want to throw when we don't know what's going on